CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+## [0.6] - 2017-11-10
+### Changed
+- Use authentic exceptions as defined in python-ldap instead of exceptions
+  defined internally.
+- Add version.py to store version number.
+- Start using isinstance() for type checking for Python3 compatibility.
```

Comment:
This should probably say "Added support for Python 3", since that's what users care about (the result, not how we did it).